app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -6,18 +6,18 @@ import notesApi from "apis/notes";
 
 const DeleteAlert = ({
   refetch,
+  selectedNote,
+  setSelectedDeleteNote,
   onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
 }) => {
   const [deleting, setDeleting] = useState(false);
 
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      await notesApi.destroy({ ids: selectedNote.id });
```

Comment:
The prop is named `ids`, that is plural, but you are passing in a string as its value. It doesn't make sense.